TKGame/Animations/ScreenTransistion.cs

```diff
@@ -22,53 +9,65 @@ public class ScreenTransition
-            sourceRectangles[4] = new Rectangle(spriteX + (buffer*2), spriteY + (buffer * 2), spriteX, spriteY);
-            sourceRectangles[5] = new Rectangle((spriteX  * 2) + (buffer * 2), spriteY + (buffer * 2), spriteX, spriteY);
-            sourceRectangles[6] = new Rectangle(buffer, (spriteY *2) + (buffer), spriteX, spriteY);
-            sourceRectangles[7] = new Rectangle((spriteX + buffer) + 2, (spriteY + (buffer * 2)) * 2, spriteX, spriteY);
-            sourceRectangles[8] = new Rectangle((spriteX * 2) + (buffer * 2), (spriteY + (buffer * 2)) * 2, spriteX, spriteY);
-            sourceRectangles[9] = new Rectangle(buffer, (spriteY * 3) + (2 * buffer) , spriteX, spriteY);
-            sourceRectangles[10] = new Rectangle((spriteX + buffer) + 2, (spriteY * 3) + (2 * buffer), spriteX, spriteY);
-            sourceRectangles[11] = new Rectangle((spriteX * 2) + (buffer * 2), (spriteY * 3) + (6 * buffer), spriteX, spriteY);
-            sourceRectangles[12] = new Rectangle(buffer, (spriteY + buffer) * 4, spriteX, spriteY);
-            sourceRectangles[13] = new Rectangle((spriteX + buffer) + 2, (spriteY + buffer) * 4, spriteX, spriteY);
-            sourceRectangles[14] = new Rectangle((spriteX * 2) + (buffer * 2), (spriteY + buffer) * 4, spriteX, spriteY);
-            sourceRectangles[15] = new Rectangle(buffer, (spriteY + buffer) * 5, spriteX, spriteY);
-            sourceRectangles[0] = new Rectangle((spriteX + buffer) + 2, (spriteY +buffer) * 5 + 16, spriteX, spriteY -15);
+            sourceRectangles     = new Rectangle[16];
+
+            sourceRectangles[0]  = new Rectangle(col[0], row[0], spriteX, spriteY);
+            sourceRectangles[1]  = new Rectangle(col[1], row[0], spriteX, spriteY);
+            sourceRectangles[2]  = new Rectangle(col[2], row[0], spriteX, spriteY);
+
+            sourceRectangles[3]  = new Rectangle(col[0], row[1], spriteX, spriteY);
+            sourceRectangles[4]  = new Rectangle(col[1], row[1], spriteX, spriteY);
+            sourceRectangles[5]  = new Rectangle(col[2], row[1], spriteX, spriteY);
+            
+            sourceRectangles[6]  = new Rectangle(col[0], row[2], spriteX, spriteY);
+            sourceRectangles[7]  = new Rectangle(col[1], row[2], spriteX, spriteY);
+            sourceRectangles[8]  = new Rectangle(col[2], row[2], spriteX, spriteY);
+            
+            sourceRectangles[9]  = new Rectangle(col[0], row[3], spriteX, spriteY);
+            sourceRectangles[10] = new Rectangle(col[1], row[3], spriteX, spriteY);
+            sourceRectangles[11] = new Rectangle(col[2], row[3], spriteX, spriteY);
+          
+            sourceRectangles[12] = new Rectangle(col[0], row[4], spriteX, spriteY);
+            sourceRectangles[13] = new Rectangle(col[1], row[4], spriteX, spriteY);
+            sourceRectangles[14] = new Rectangle(col[2], row[4], spriteX, spriteY);
 
+            sourceRectangles[15] = new Rectangle(buffer, row[5], spriteX, spriteY);
```

Comment:
I feel like this can be cleaned up by having something like 
```csharp
sourceRectangles[16] = new Rectangle[16]
{
    { col[0], row[0], spriteX, spriteY },
    etc...
};
```